src/OpenLoco/src/Windows/LandscapeGeneration.cpp

```diff
@@ -159,23 +160,57 @@ namespace OpenLoco::Ui::Windows::LandscapeGeneration
     {
         enum widx
         {
-            start_year = Common::widx::tab_industries + 1,
+            groupGeneral = Common::widx::tab_industries + 1,
+            start_year,
             start_year_down,
             start_year_up,
-            generator,
-            generator_btn,
+            heightMapBox,
+            heightMapDropdown,
+
+            groupGenerator,
+            change_heightmap_btn,
+            terrainSmoothingNum,
+            terrainSmoothingNumDown,
+            terrainSmoothingNumUp,
             generate_when_game_starts,
+
             generate_now,
```

Comment:
I couldn't find where the exact problem is, but when you toggle the "Only generate random landscape when game starts" button, the functionality works but checkbox never draws itself checked.

eg the option is checked here:
![image](https://github.com/OpenLoco/OpenLoco/assets/7483209/9070bda9-baa0-448e-9c41-74d798ec9e3b)

and unchecked here
![image](https://github.com/OpenLoco/OpenLoco/assets/7483209/30ee75c4-a153-473a-94e9-c7c07d13cec6)

Reply:
Second issue with this - when the option is unchecked and you click "Generate new landscape", the landscape generates and the option is *not* changed to unchecked. I think better behaviour is to disable the "Generate new landscape" button when this checkbox is checked.